### PR TITLE
Return ScheduledActivity in same TimeZone as in the request

### PR DESCRIPTION
**Pull request by liujoshua**

### CODE REVIEW

File: app/org/sagebionetworks/bridge/dao/ScheduledActivityDao.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import java.util.List;
 
+import org.joda.time.Chronology;
```
Comment (alxdarksage) [inline app/org/sagebionetworks/bridge/dao/ScheduledActivityDao.java]: I don't think you need Chronology, at least elsewhere I've called dateTime.getZone() without issue.

File: app/org/sagebionetworks/bridge/services/ScheduledActivityService.java [hunk 2]
```diff
@@ -125,8 +129,13 @@ final void setSurveyService(SurveyService surveyService) {
             throw new BadRequestException(EITHER_BOTH_DATES_OR_NEITHER);
         }
 
+        DateTimeZone timezone = scheduledOnStart.getChronology().getZone();
```
Comment (alxdarksage) [inline app/org/sagebionetworks/bridge/services/ScheduledActivityService.java]: Can you just call scheduledOnStart.getZone() ?

**Review by alxdarksage — CHANGES_REQUESTED** [review]
Only one minor comment about use of getChronology() most places...

File: app/org/sagebionetworks/bridge/services/ScheduledActivityService.java (hunk 2 above)
Comment (liujoshua) [reply app/org/sagebionetworks/bridge/services/ScheduledActivityService.java]: Turns out, yes

**Pull request merged**